src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java

```diff
@@ -168,7 +171,15 @@ public void scenario1() throws IOException {
         setAuth(request, "smith123");
         try (final CloseableHttpResponse response = execute(request)) {
             assertEquals(HttpStatus.SC_OK, getStatus(response));
+            for (final Header header : response.getHeaders("Link")) {
+                logger.debug("Link header: {}", header.getValue());
+                if (header.getValue().equals(aclLink)) {
+                    return;
+                }
+            }
+            assertEquals("Missing Link Header " + aclLink.toString(), 1, 0);
```

Comment:
@whikloj in this case, you can just roll back the change on this line from #47 so you get:

    try (final CloseableHttpResponse response = execute(request)) {
        assertEquals(HttpStatus.SC_OK, getStatus(response));
        // the rest of your code from your PR
    }

It's fine to use the `try` block if you actually need to inspect the response. For merely checking the status code it was unnecessary.